package.json

```diff
@@ -209,26 +209,17 @@
     "@types/sprintf-js": "^1.1.2",
     "@types/url-parse": "^1.4.8",
     "@types/uuid": "^7.0.0",
-    "@typescript-eslint/eslint-plugin": "^5.36.2",
-    "@typescript-eslint/parser": "^5.36.2",
     "@welldone-software/why-did-you-render": "^8.0.1",
+    "add": "^2.0.6",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^29.6.3",
     "body-parser": "^1.18.2",
     "buffer": "^6.0.3",
     "crypto-browserify": "^3.12.0",
     "esbuild-loader": "^2.20.0",
-    "eslint": "^8.57.0",
-    "eslint-config-standard-kit": "0.15.1",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-local-rules": "^3.0.2",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.1.4",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.21.5",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-plugin-react-native": "^3.8.1",
-    "eslint-plugin-simple-import-sort": "^6.0.1",
+    "eslint": "^9.31.0",
+    "eslint-config-standard-kit": "./eslint-config-standard-kit-0.15.1.tgz",
```

Comment:
Broken yarn install. Are we trying to have this file committed for some reason or did we mean to reference the registry?